templates/landscape/install.html

```diff
@@ -81,90 +88,91 @@ <h2 class="p-heading--5 u-sv3">The 3 ways of deploying landscape</h2>
       </div>
     </div>
   </div>
+  <div class="p-strip is-shallow u-no-padding-bottom u-hide--large u-hide--medium">
+    <div class="row">
+      <hr class="is-fixed-width">
+      <div class="u-fixed-width">
```

Comment:
```suggestion
      <hr class="is-fixed-width">
      <div class="u-fixed-width">
```
You don't need to wrap a `u-fixed-width` in a row